packages/web-components/src/components/pictogram-item/pictogram-item.ts

```diff
@@ -7,12 +7,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { customElement, html, LitElement, property } from 'lit-element';
+import { customElement, html } from 'lit-element';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
 import settings from 'carbon-components/es/globals/js/settings';
+import DDSContntItem from '../content-item/content-item';
```

Comment:
@asudoh a misspelling here for `DDSContntItem`

Reply:
Good catch @jeffchew thanks! Fixed.